WordPressAuthenticator.podspec

```
@@ -1,20 +1,23 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressAuthenticator"
   s.version       = "1.36.0-beta.2"
-  s.summary       = "WordPressAuthenticator implements an easy and elegant way to authenticate your WordPress Apps."
 
+  s.summary       = "WordPressAuthenticator implements an easy and elegant way to authenticate your WordPress Apps."
   s.description   = <<-DESC
                     This framework encapsulates everything required to display the Authentication UI
                     and perform authentication against WordPress.com and WordPress.org sites.
 
-                    Plus: WordPress.com *signup*  is supported.
+                    Plus: WordPress.com *signup* is supported.
                     DESC
 
-  s.homepage      = "https://github.com/wordpress-mobile/WordPressAuthenticator-iOS"
-  s.license       = "GPLv2"
-  s.author        = { "WordPress" => "mobile@automattic.com" }
+  s.homepage      = "http://apps.wordpress.com"
```

This is for consistency between all the pods, and because the url to the GitHub repo is already available as part of other metadata (especially `s.source.git`)

---

Is there a reason to use HTTP over HTTPS, maybe a CocoaPods convention? If you open the HTTP URL, it redirects to the HTTPS one.

```suggestion
  s.homepage      = "https://apps.wordpress.com"
```

---

No, only because it was the url that was used in other podspecs of ours I copy/paste that from. Good catch!